### PR TITLE
View Draft Transfer Evidence Note Bug Fixes

### PR DESCRIPTION
**Pull request by NateHogan**
This PR contains fixes for the following
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/46209
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/46210
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/46211
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/46213
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/46214

This PR also addresses comments made in this PR
https://github.com/DEFRA/prsd-weee/pull/873

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml [hunk 1]
```diff
@@ -50,19 +51,15 @@
         {
             var item = categoryCount;
 
-            @Html.Hidden("categoryid_to_calculate" + categoryCount, Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId)
-            @Html.HiddenFor(m => m.TotalCategoryValues[item].CategoryDisplay)
-            var receivedId = "Received" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
-            var reusedId = "Reused" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
+            //var receivedId = "Received" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
+            //var reusedId = "Reused" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml]: if that's not going to be needed, would you mind deleting it?

File: src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml [hunk 2]
```diff
@@ -79,10 +76,10 @@
         <caption class="govuk-table__caption govuk-visually-hidden">Transfer tonnages for @aatf.AatfName</caption>
         <thead class="govuk-table__head">
             <tr class="govuk-table__row">
-                <th class="govuk-table__header govuk-!-text-align-left" style="width: 10%;">
+                <th class="govuk-table__header govuk-!-text-align-left" style="width: 15%;">
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml:76]: can you not use any govuk class for setting the width, one-quarter and etc?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml (hunk 1 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml]: I think deleted now

File: src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml (hunk 2 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml:76]: The one-quarter rule seems to float left which ruins the formatting of the table
There are no set ones for 15%/35% and unsure if needed to create a CSS rule for this as unlikely to be used elsewhere

This specific formatting was taken from the Transfer Tonnages page

**Review by meglenapnovoselska — APPROVED** [review]

**Comment by JohnPadgettCivica** [comment]
Just need to resolve the merge conflicts...

**Pull request merged**